pkgs/applications/audio/gwc/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkgconfig
+, alsaLib
+, libpulseaudio
+, gtk2
+, hicolor-icon-theme
+, libsndfile
+, fftw
+, perl
+, xdg_utils
+, vorbis-tools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gwc";
+  version = "0.22-04";
+
+  src = fetchFromGitHub {
+    owner = "AlisterH";
+    repo = pname;
+    rev = version;
+    sha256 = "0xvfra32dchnnyf9kj5s5xmqhln8jdrc9f0040hjr2dsb58y206p";
+  };
+
+  buildInputs = [
+    alsaLib
+    libpulseaudio
+    gtk2
+    hicolor-icon-theme
+    libsndfile
+    fftw
+    perl
+    xdg_utils
+    vorbis-tools
```

Comment:
Not sure how are these used but unless their paths are hardcoded by the build system, you will not see them at runtime. You would need to patch the files like here https://github.com/NixOS/nixpkgs/blob/c60e2bee7c514a422ebf74c766e9e18c936ad845/pkgs/applications/misc/orca/default.nix#L47-L55, or add them to `PATH` using `wrapProgram`.

Reply:
These where optional, for opening the online help (still works), and for batch processing (unneeded).